solutions/skamble89/rectangle.java

```diff
@@ -0,0 +1,11 @@
+package solutions.skamble89;
+
+public class rectangle extends shape {

```

Comment:
We generally follow these naming practices in Java. 

https://www.javatpoint.com/java-naming-conventions